VotingApplication/VotingApplication.Web/Controllers/LoginController.cs

```diff
@@ -18,5 +18,10 @@ public ActionResult ForgottenPassword()
         {
             return View();
         }
+
+        public ActionResult EmailNotConfirmed()
```

Comment:
I wonder if these views would be easier to find/organise if they were in their own folder/controller